runelite-client/src/main/java/net/runelite/client/plugins/implings/ImplingMinimapOverlay.java

```diff
@@ -73,7 +73,16 @@ public Dimension render(Graphics2D graphics)
 			if (config.showName())
 			{
 				Point textLocation = new Point(impLocation.getX() + 1, impLocation.getY());
-				OverlayUtil.renderTextLocation(graphics, textLocation, imp.getName(), color);
+
```

Comment:
Remove this newline?

Reply:
You may have removed the wrong one in 8162820? I think that change is fine also, but I suggested this to keep variable assignment separated from other function calls, and not have random line breaks separating variables.

Reply:
I couldn't figure out which line you're talking about? I scanned it and didn't see any white space, except the one I already submitted a commit for.